src/msw/notebook.cpp

```diff
@@ -499,6 +499,20 @@ void wxNotebook::SetTabSize(const wxSize& sz)
     ::SendMessage(GetHwnd(), TCM_SETITEMSIZE, 0, MAKELPARAM(sz.x, sz.y));
 }
 
+wxRect wxNotebook::GetTabRect(size_t n) const
+{
+    wxRect r;
+
+    if (GetPageCount() > 0)
+    {
+        RECT rect;
+        if (TabCtrl_GetItemRect(GetHwnd(), n, &rect))
+            wxRectFromRECT(rect);
```

Comment:
```suggestion
            r = wxRectFromRECT(rect);
```

BTW, I meant to change the method argument name from `n` to `page` everywhere it was used (similarly for typos in the comments).

Reply:
Fixed